### ISSUE TITLE
feat: Add info for hiding the shorts button

### ISSUE THREAD

**Issue by KobeW50** [revanced/revanced-patches#2267]
### Application

YT ReVanced 

### Issue

Many users are confused and think that if they enable the "Hide shorts" toggle in the Shorts component settings that it will also hide the shorts button in the navigation bar.

I have seen several posts caused by this confusion.

### Patch

Add a box in the Shorts components settings menu with text stating:

"The Shorts button can be toggled in Settings > ReVanced > Navigation button settings. The Navigation Buttons patch is required.

Here is an demonstration. The demonstration doesnt include all of the text because of my poor editing skills.

![Screenshot_20230525_162207](https://github.com/revanced/revanced-patches/assets/84587632/a7823480-d1aa-4142-83a9-5051e803eca4)

### Motivation

So people aren't confused.

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by oSumAtrIX** [comment]
Or use the title "Hide shorts in feed"

**Comment by KobeW50** [comment]
Also, please lmk if something similar to this should be considered a bug or a patch request.

**Comment by oSumAtrIX** [comment]
Depends, I'd consider this an improvement of a feature. Hence it is not a patch and not a bug request.

**Comment by KobeW50** [comment]
> Depends, I'd consider this an improvement of a feature. Hence it is not a patch and not a bug request.

Perhaps a new category can be made? Maybe "Other"?

Also, the discussion and documentation can be removed, because those are empty

![SmartSelect_20230525_164709_Chrome.jpg](https://github.com/revanced/revanced-patches/assets/84587632/cc6030dc-c928-40ef-b7b4-b2dd7ed45e46)

**Comment by oSumAtrIX** [comment]
The discussions and documentation shouldn't be removed as they are closely related, a better description can be used though

**Comment by KobeW50** [comment]
> The discussions and documentation shouldn't be removed as they are closely related, a better description can be used though

Nvm about the documentation. But the discussion only brings you to this discussions page, where the only post tells you to make patch requests in the ReVanced patches repo, which the user was already trying to do.

![Screenshot_20230525_165750_GitHub.jpg](https://github.com/revanced/revanced-patches/assets/84587632/bf8b68ec-67cb-4233-9e07-91eec8b680d2)

**Comment by oSumAtrIX** [comment]
Looks like a bug on your end. Works here:

![image](https://github.com/revanced/revanced-patches/assets/13122796/608a5653-f197-4e78-8def-f1873264ac0b)

**Comment by KobeW50** [comment]
> Or use the title "Hide shorts in feed"

This would help a bit, but it won't inform users where the option to hide the shorts button is.

**Comment by KobeW50** [comment]
> There was previous discussion to change some of the default settings (so by default shorts and the shorts navigation button are present) with the goal of providing a custom YouTube that gives _more_ features and not less.  But so far no changes have been made.

True. The only issue I could find regarding this is #744, which was a long time ago. Should I make a new issue, that goes through all of the toggles that should maybe be changed?

**Comment by oSumAtrIX** [comment]
There are as many people who want the button gone as there are people who want it; either way will cause the same amount of confusion.

**Comment by KobeW50** [comment]
> Yeah either make a new PR, or go straight to making a PR and any potential changes can be discussed and decided there.

Idrk how pull requests work, so I'm just gonna open an issue.

**Comment by oSumAtrIX** [comment]
> But should the default be to remove it?

Both ways are possible and are equally valid. Someone that included the patch to remove it expects it to be gone. ReVanced comes with a (partially opinionated) default configuration that may satisfy most people; for example, ads are disabled by default. In either case, people will have to go to the settings to toggle it. Ultimately, it does not matter much, so you can leave it disabled or enable it by default. Satisfies either group of people.
